opwen_email_server/actions.py

```diff
@@ -212,7 +215,8 @@ def _decode_attachments(cls, email: dict) -> dict:
 
 
 class ReceiveInboundEmail(_Action):
-    def __init__(self, auth: AzureAuth, raw_email_storage: AzureTextStorage, next_task: Callable[[str], None]):
```

Comment:
Nit: looks like this line doesn't need to change anymore.

Reply:
for the auth I think I had to add Union[AzureAuth, NoAuth] to satisfy the type checker and keep the build from failing

Reply:
Good catch. I didn't spot that. Thanks.